CRM/Event/Tokens.php

```diff
@@ -215,16 +210,12 @@ protected function getEventTokenValues(int $eventID = NULL): array {
         'state_province' => $event['loc_block_id.address_id.state_province_id:label'],
         'postal_code' => $event['loc_block_id.address_id.postal_code'],
       ]);
-      $tokens['location']['text/html'] = nl2br(trim($tokens['location']['text/plain']));
```

Comment:
Theoretically, we've lost a trim() for the address block, but doesn't seem to be needed in testing (note that the token is not currently in use as it is not fully implemented (supplemental addresses, etc))